.gitignore

```diff
@@ -141,6 +141,7 @@ publish/
 # NuGet Packages
 *.nupkg
 **/packages/*
+/packages/*
```

Comment:
My diff tool doesn't understand the ** syntax the same way that git does, so it shows me tons of useless diffs under /packages.  My change adds all the files in the directory "packages" at the root of the enlistment to the ignore list.  This should be safe as it's just a special case of **/packages/*.